Src/Tools/BuildGeneratedArgumentTranslator/Keywords.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Copyright 2019 Google LLC
```

Comment:
nit: 2024?

Reply:
Unsure. It's a copy of a file from elsewhere. Happy to do so - I never know in this situation.